src/main/java/mate/academy/Main.java

```diff
@@ -15,11 +16,11 @@ public static void main(String[] args) {
 
         for (int i = 0; i < userIds.length; i++) {
             String userId = userIds[i];
+
             futures[i] = asyncRequestProcessor.processRequest(userId)
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
Ensure that `executor.shutdown()` is called after all tasks have completed. The current implementation does this correctly, but it's important to remember that `shutdown()` will not wait for tasks to complete. If you need to wait for all tasks to finish, consider using `executor.awaitTermination()` after `shutdown()`.